cli/cmd/root.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"html/template"
+	"os"
+	"strings"
+)
+
+type Metadata struct {
+	ProjectName   string
+	SourcePackage string
+}
+
+var rootCmd = &cobra.Command{
+	Use:   "jinycli",
+	Short: "JinyFramework CLI",
+	Long: `Lightweight, modern, simple Java web framework for rapid development in the API era`,
+	Run: func(cmd *cobra.Command, args []string) {
+		Intro()
+
+		projectNamePrompt := promptui.Prompt{
+			Label:    "Project Name",
+			Default:  "test",
+		}
+		sourcePackagePrompt := promptui.Prompt{
+			Label:    "Source Package",
+			Default:  "com.yourcompany",
+		}
+		projectNameGot, _ := projectNamePrompt.Run()
+		sourcePackageGot, _ := sourcePackagePrompt.Run()
+
+		path, _ := os.Getwd()
+		path = path +  string(os.PathSeparator)
+		addPath := projectNameGot + string(os.PathSeparator)
+		path = path + addPath
+		m := Metadata{ProjectName: projectNameGot, SourcePackage: sourcePackageGot}
+
+		if _, err := os.Stat(path); os.IsNotExist(err) {
+			_ = os.Mkdir(path, os.ModePerm)
+		}
+
+		// Create source package path dir
+		sourcePackages := strings.Split(m.SourcePackage, ".")
+		sourcePackagesPath := ""
+		for i := 0; i < len(sourcePackages); i++ {
+			sourcePackagesPath += sourcePackages[i] + string(os.PathSeparator)
+		}
+
+		// Create src main
+		_ = os.MkdirAll(path+string(os.PathSeparator)+"src"+string(os.PathSeparator)+"main"+string(os.PathSeparator)+"java"+string(os.PathSeparator)+sourcePackagesPath, os.ModePerm)
```

Comment:
These paths should be refactored, maybe save os.PathSeparator to variable, create constants for path, and use sprintf to fill in params